styles.css

```diff
@@ -0,0 +1,949 @@
+:root {
+  --color-black: #12130f;
+  --color-gray-dark: #4d4d4d;
+  --color-white: #ffffff;
+  --color-cream: #f6e6cd;
+  --color-gray-medium: #333333;
+  --color-light-white: #fffaf1;
+  --color-black-solid: #000000;
+  --color-orange-light: #ffa613;
+  --color-orange: #f66741;
+  --color-purple: #7a86f2;
+  --color-orange-dark: #f7961e;
+
+  --padding-x: 50px;
+  --header-height: 80px;
+}
+
```

Comment:
you can add default props 

*:{
 margin: 0;
  padding: 0;
  box-sizing: border-box;
  font-family: "M PLUS Code Latin", monospace;
  }
  
  so you dont need to set the common props in all classes

Reply:
fixed